src/main/java/com/example/LooseCouplingDemo/mapper/MessagesMapper.java

```diff
@@ -0,0 +1,27 @@
+package com.example.LooseCouplingDemo.mapper;
+
+import com.example.LooseCouplingDemo.dto.AddMessagesDTO;
+import com.example.LooseCouplingDemo.model.Messages;
+import org.mapstruct.InheritInverseConfiguration;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring", uses = UsersMapper.class)
+public interface MessagesMapper {
+
+    @Mapping(source = "content", target = "content")
+    @Mapping(source = "type", target = "type")
+    Messages DTOToEntity(AddMessagesDTO addMessagesDTO);
```

Comment:
```suggestion
    Messages DtoToEntity(AddMessagesDTO addMessagesDTO);
```
Names should be in camel case

Reply:
Done